etc/containerpilot.json

```diff
@@ -8,7 +8,16 @@
       "port": 80,
       "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
       "poll": 10,
-      "ttl": 25
+      "ttl": 25,
+      "interfaces": ["eth0"]
+    },
+    {
+      "name": "nginx-public",
```

Comment:
This has some backwards-incompatible implications that might be avoided if we set `nginx` as public and add a `nginx-private` service.